aws_helper/config.go

```diff
@@ -83,10 +83,33 @@ func CreateAwsSession(config *AwsSessionConfig, terragruntOptions *options.Terra
 	} else if terragruntOptions.IamRole != "" {
 		sess.Config.Credentials = stscreds.NewCredentials(sess, terragruntOptions.IamRole, credentialsOptFn)
 	}
+	return sess, nil
+}
+
+// Returns an AWS session object for the given config region (required), profile name (optional), and IAM role to assume
```

Comment:
NIT: Are profile name + IAM role passed in via `terragruntOptions`? If so, we should say so explicitly.

Reply:
Updated in 861668f